scss/modules/_site-header.scss

```diff
@@ -1,7 +1,21 @@
 // Site Header
-// 
+//
 // Styleguide modules.site-header
 
+// Staging banner
+.banner {
+  background-color: $navy;
+  color: $cream;
+  font-family: $sans-serif;
+  padding: 1rem 0;
+
+  a {
+    color: $cream;
```

Comment:
Yeah. I was trying to do that but I'm fine to lose it. It strikes me as really unimportant.